src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", ()=> {
+    const { asFragment } = render(<Search />);
+
+    it("renders correctly", ()=>{
```

Comment:
Could add a test here for checking the function is called when the button is clicked